spec/support/activerecord.rb

```diff
@@ -25,7 +25,3 @@
 ActiveRecord::Base.establish_connection(dbconfig.merge(:database => database))
 
 ActiveRecord::Migration.verbose = false
-
-if ActiveRecord::VERSION::STRING >= '4.2'
-  ActiveRecord::Base.raise_in_transactional_callbacks = true
```

Comment:
Without this, ActiveRecord 4.2 produces tons of deprecation warnings.
https://travis-ci.org/carrierwaveuploader/carrierwave/jobs/166306676